bazel/grpc_build_system.bzl

```diff
@@ -87,6 +89,11 @@ def grpc_cc_library(
     linkopts = if_not_windows(["-pthread"])
     if use_cfstream:
         linkopts = linkopts + if_mac(["-framework CoreFoundation"])
+    # Temporary hack for GRPC_USE_ABSL {
```

Comment:
If you have a temporary hack, please add a TODO with an associated Github issue.

Reply:
Done.